onmetal-controller-manager/client/client.go

```diff
@@ -27,6 +27,10 @@ const (
 	MachineSpecNetworkInterfaceNamesField = "machine-spec-network-interface-names"
 	MachineSpecVolumeNamesField           = "machine-spec-volume-names"
 
+	AliasPrefixNetworkName = "aliasprefix-network-name"
+
+	LoadBalancerNetworkName = "loadbalancer-network-name"
```

Comment:
All our fields should have a suffix `Field` in the name to allow the reader to easily distinguish what's in the `client` package.